spec/controllers/api/collectivities_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+describe Api::CollectivitiesController, type: :controller do
+  describe "GET index" do
+    let(:collectivities) { create_list(:collectivity, 5) }
+
+    before do
+      collectivities
+    end
```

Comment:
```suggestion
    let!(:collectivities) { create_list(:collectivity, 5) }
    let!(:invalid_collectivity) { create(:collectivity, status: 'inactive')
```

Et en dessous tu testes le bon nombre